build.gradle

```diff
@@ -24,9 +34,9 @@ dependencies {
     compile "com.google.code.gson:gson:2.2.4"
     compile "com.google.guava:guava:19.0"
     compile "org.slf4j:slf4j-api:1.7.7"
-    compile "com.launchdarkly:okhttp-eventsource:0.1.0"
+    compile group: "com.launchdarkly", name: "okhttp-eventsource", version: "0.1.3-SNAPSHOT", changing: true
```

Comment:
From here: https://discuss.gradle.org/t/how-to-get-gradle-to-download-newer-snapshots-to-gradle-cache-when-using-an-ivy-repository/7344

In short it goes with the addition on lines 8-12. We're instructing gradle to always go look in a repo for the latest version of this dep. When this is pinned to a stable 1.0.0 it won't be needed, but is very handy for local development and potentially build/deploy systems that that might have older snapshots cached.

I am not a gradle expert, so this might not be the best way to do it.